include/dlaf/common/unwrap.h

```diff
@@ -12,6 +12,7 @@
 /// @file
 
 #include <functional>
+#include <type_traits>
```

Comment:
```suggestion
```

Reply:
Sorry, that's a very good catch! I thought it was just a leftover from moving `consume_rvalues` from `unwrap.h` into a separate header.

Would you mind adding it back? It belongs there, as does `utility` (for `forward`). Separate PR is also fine.